### DISCUSSION TITLE
Customizable Exports

### DISCUSSION THREAD

**Discussion by DWrightAnimation** [keiwando/toonsquid#497] [Ideas]
Callipeg has the ability to name each image sequence image the way you want it to be named, like if you needed to export an image sequence and import it into OpenToonz, you can make it's name like "Example.0001", but in ToonSquid, it automatically exports as "Example_01". Being able to export an image sequence anyway I want is useful if I need to do a certain thing in a different app, such as exporting the image sequence with a transparent background, then importing it into OpenToonz, then using the Erode/Dilate effect to make my animation look like something from Fairly OddParents or The Grim Adventures of Billy and Mandy.

**Comment by Cat-Butler** [comment]
While an in-built solution is definitely more convenient, as an interim workaround:

If you name your export file Example. (with the dot/full stop/period at the end), you will get Example._0001, Example._0002, etc (and as you no doubt already know, there's a hidden _.png_ at the end, usually with options in your file browser(s) to show these or not).

If you're opening it in OpenToonz, you must be moving to a desktop context at some stage, which opens up all sorts of renaming possibilities. Without downloading a renamer utility:

PowerShell example to take out those underscores in Windows 10/11:
`Get-ChildItem -Path "." -Recurse -Include "*.png" | Rename-Item -NewName { $_.Name -replace "_","" }`

It's probably even easier to read in a similar one-line macOS terminal script, but I don't currently have one close by to check my work, so I'm hesitant to post an untested script. Let me know if you're interested and I'll put one together in the next night or two.
